core/Tracker/Settings.php

```diff
@@ -120,6 +122,10 @@ protected function getConfigHash(Request $request, $os, $browserName, $browserVe
             $configString .= $request->getIdSite();
         }
 
+        if (PrivacyManager::shouldAnonymiseFingerprint()) {
+            $configString .= Date::now()->setTimezone(Site::getTimezoneFor($request->getIdSite()))->toString();
```

Comment:
todo: generate random string per site per day or so to anonymise it more

Reply:
:+1:  and quite a long string so it's not possible to guess if one knew all the other data points